demoapp/demo_app_with_12bills.py

```diff
@@ -2,225 +2,337 @@
 This code only implements text summarization, category selection and tagging for 12 bills
 Uses OpenAIEmbeddings and Vectorstore to split the MGL text into chunks, storing it and performing vector search
 Changed the code to use selected bill number from the app to access the MGL text for that bill (original code used combined MGL text from all the extracted sections for all 12 bills)

```

Comment:
You don't have to do everything in one PR, but eventually we will want to have modularity here such that the repeated functions and constants from demo_app.py are imported rather than redefined for this second app. The code for the second app should be very small, since it is mostly reusing logic.

Reply:
Ok, I can modify the files to have only one file for the main set of functions, and two other files to run the app, one for all the bills and the other one for 12 bills.